tests/session/cmd/test_show_config.py

```diff
@@ -138,6 +138,7 @@ def test_pass_env_config_default(tox_project: ToxProjectCreator, stdout_is_atty:
         + (["TERM"] if stdout_is_atty else [])
         + (["TMP", "USERPROFILE"] if is_win else ["TMPDIR"])
         + ["VIRTUALENV_*", "http_proxy", "https_proxy", "no_proxy"]
+        + (["windir"] if is_win else [])
```

Comment:
should be all capital

Reply:
I think I've addressed this; amended and force-pushed for a clean history.